torch/csrc/profiler/util.cpp

```diff
@@ -292,6 +292,18 @@ std::string strListToStr(const std::vector<std::string>& types) {
     return "[" + rc + "]";
   }
 }
+std::string ivalueToStr(const c10::IValue& val) {
+  if (val.isNone()) {
+    return "\"None\"";
+  } else {
+    std::stringstream ss;
+    ss.str("");
+    ss << "\"";
+    ss << val;
```

Comment:
do we need to sanitize this value?

Reply:
ok sorry, changed my mind haha - probably fine as it was before; I think this probably gets sanitized on the kineto side?

if you don't mind, would you be able to sanity check whatever choice you make with a test? I'm just slightly over-concerned about this because I think we've had SEVs before from broken traces due to unsanitized text in the json

Reply:
I think kineto just sanitizes for backslashes and new lines. We might wanna keep this as is and then later on try to unify all the sanitizations that happen in different areas in the code with a follow up.

Reply:
Will add a test still